variants/Generic_F410Cx/variant.h

```diff
@@ -99,9 +99,8 @@ extern "C" {
 #define PIN_WIRE_SCL            PB8
 
 // Timer Definitions
-// Use TIM6/TIM7 when possible as servo and tone don't need GPIO output pin
 #define TIMER_TONE              TIM6
-#define TIMER_SERVO             TIM7
+#define TIMER_SERVO             TIM5
```

Comment:
I would prefer TIM11 which is the most basic timer (after TIM6/7). So that we leave more complex timers for user.

Reply:
done